docs/ref/csrf.txt

```diff
@@ -92,6 +92,9 @@ protection for your views as outlined above.
     The CSRF token cookie is named ``csrftoken`` by default, but you can control
     the cookie name via the :setting:`CSRF_COOKIE_NAME` setting.
 
+    The CSRF header name is ``HTTP_X_CSRFTOKEN`` by default, but you can
```

Comment:
A few words about `X-CSRFToken` vs. `HTTP_X_CSRFTOKEN` would help.

Most likely you can steal them from one of the many places where the docs talk about HTTP headers.